soh/src/overlays/actors/ovl_Bg_Gjyo_Bridge/z_bg_gjyo_bridge.c

```diff
@@ -106,25 +106,25 @@ void BgGjyoBridge_TriggerCutscene(BgGjyoBridge* this, PlayState* play) {
                 }
                 break;
             case RO_BRIDGE_STONES:
-                if (CheckStoneCount() >= bridgeStoneCount) {
+                if ((CheckStoneCount() + CheckBridgeRewardCount()) >= bridgeStoneCount) {
                     this->actionFunc = BgGjyoBridge_SpawnBridge;
                     func_800F595C(NA_BGM_BRIDGE_TO_GANONS);
                 }
                 break;
             case RO_BRIDGE_MEDALLIONS:
-                if (CheckMedallionCount() >= bridgeMedallionCount) {
+                if ((CheckMedallionCount() + CheckBridgeRewardCount()) >= bridgeMedallionCount) {
                     this->actionFunc = BgGjyoBridge_SpawnBridge;
                     func_800F595C(NA_BGM_BRIDGE_TO_GANONS);
                 }
                 break;
             case RO_BRIDGE_DUNGEON_REWARDS:
-                if ((CheckMedallionCount() + CheckStoneCount()) >= bridgeRewardCount) {
+                if ((CheckMedallionCount() + CheckStoneCount() + CheckBridgeRewardCount()) >= bridgeRewardCount) {
                     this->actionFunc = BgGjyoBridge_SpawnBridge;
                     func_800F595C(NA_BGM_BRIDGE_TO_GANONS);
                 }
                 break;
             case RO_BRIDGE_DUNGEONS:
-                if (CheckDungeonCount() >= bridgeDungeonCount) {
+                if ((CheckDungeonCount() + CheckBridgeRewardCount()) >= bridgeDungeonCount) {
```

Comment:
do we want greg to count as a wildcard for dungeons? i understand for dungeon rewards, but having greg count as completing a dungeon seems odd

Reply:
I mean, you were the one who suggested it. https://github.com/HarbourMasters/Shipwright/pull/2549#pullrequestreview-1316690181

Reply:
I don't one way of another. I never use the dungeon completion option. But those who do might like having the option.